build.sbt

```diff
@@ -23,7 +23,7 @@ import uk.gov.hmrc.SbtAutoBuildPlugin
 
 val appName = "vat-correspondence-details-frontend"
 
-val bootstrapPlayVersion       = "3.0.0"
```

Comment:
Could you please also upgrade the govUkTemplate, playUi and playLanguage versions? (e.g. as in deregister-vat-frontend or vat-designatory-details-frontend)